### PR TITLE
add privacy information about aclk connection

### PR DESCRIPTION
**Pull request by OdysLam**

### CODE REVIEW

File: aclk/README.md [hunk 1]
```diff
@@ -129,3 +129,28 @@ If you changed the runtime setting in your `var/lib/netdata/cloud.d/cloud.conf`
 Restart your Agent and [claim your node](/claim/README.md#how-to-claim-a-node).
 
 [![analytics](https://www.google-analytics.com/collect?v=1&aip=1&t=pageview&_s=1&ds=github&dr=https%3A%2F%2Fgithub.com%2Fnetdata%2Fnetdata&dl=https%3A%2F%2Fmy-netdata.io%2Fgithub%2Faclk%2FREADME&_u=MAC~&cid=5792dfd7-8dc4-476b-af31-da2fdb9f93d2&tid=UA-64295674-3)](<>)
+
+# ACLK and privacy
+
+Privacy is very important to us and we firmly believe that the data of our users belong to themselves. This is why w**e actually don't store any metric data in the Netdata Cloud**.
```
Comment (ilyam8) [inline aclk/README.md:135]:
```suggestion
Privacy is very important to us and we firmly believe that the data of our users belong to themselves. This is why **we actually don't store any metric data in the Netdata Cloud**. 
```

File: aclk/README.md [hunk 2]
```diff
@@ -129,3 +129,28 @@ If you changed the runtime setting in your `var/lib/netdata/cloud.d/cloud.conf`
...
+
+The information we store in Netdata Cloud is the following (using the publicly available demo server `frankfurt.my-netdata.io` as an example) :
+- User data:
+  - user email
+- For each Netdata Agent node that is connected to Netdata Cloud:
+ - Node Name
+ - The node information shown in `/api/v1/info`. E.g. `https://frankfurt.my-netdata.io/api/v1/info`
+ - The chart metadata shown in `/api/v1/charts`. E.g. `https://frankfurt.my-netdata.io/api/v1/charts`
+ - The alarm configurations shown in `/api/v1/alarms?all`. E.g. `https://frankfurt.my-netdata.io/api/v1/alarms?all`
+ - Active alarms shown in `/api/v1/alarms`. E.g `https://frankfurt.my-netdata.io/api/v1/alarms`
```
Comment (ilyam8) [inline aclk/README.md:151]: those frankfurt links supposed to be hyper links?

[`https://frankfurt.my-netdata.io/api/v1/alarms`](https://frankfurt.my-netdata.io/api/v1/alarms)

Comment (ilyam8) [reply aclk/README.md:151]: https://github.com/netdata/netdata/blob/aclk-privacy/aclk/README.md#metadata

File: aclk/README.md [hunk 3]
```diff
@@ -129,3 +129,28 @@ If you changed the runtime setting in your `var/lib/netdata/cloud.d/cloud.conf`
 Restart your Agent and [claim your node](/claim/README.md#how-to-claim-a-node).
 
 [![analytics](https://www.google-analytics.com/collect?v=1&aip=1&t=pageview&_s=1&ds=github&dr=https%3A%2F%2Fgithub.com%2Fnetdata%2Fnetdata&dl=https%3A%2F%2Fmy-netdata.io%2Fgithub%2Faclk%2FREADME&_u=MAC~&cid=5792dfd7-8dc4-476b-af31-da2fdb9f93d2&tid=UA-64295674-3)](<>)
+
+# ACLK and privacy
```
Comment (joelhans) [inline aclk/README.md:133]:
```suggestion
## Data privacy
```

File: aclk/README.md [hunk 4]
```diff
@@ -129,3 +129,28 @@ If you changed the runtime setting in your `var/lib/netdata/cloud.d/cloud.conf`
...
+
+Privacy is very important to us and we firmly believe that the data of our users belong to themselves. This is why **we actually don't store any metric data in the Netdata Cloud**.
```
Comment (joelhans) [inline aclk/README.md:135]: Let's personalize this and stay away from abstractions like "the user." IMO that's for legal content, not documentation.

```suggestion
Privacy is very important to us. We firmly believe that your data belongs to you. This is why **we don't store any metric data in Netdata Cloud**. 
```

File: aclk/README.md [hunk 5]
```diff
@@ -129,3 +129,28 @@ If you changed the runtime setting in your `var/lib/netdata/cloud.d/cloud.conf`
...
+
+Privacy is very important to us and we firmly believe that the data of our users belong to themselves. This is why **we actually don't store any metric data in the Netdata Cloud**. 
+
+All the data that the user sees in the web browser when using Netdata Cloud, are actually streamed directly from the Netdata Agent to the Netdata Cloud dashboard. They pass through our systems, but they are not stored.
+
+We do however store a limited number of *metadata* to be able to offer the stunning visualizations and advanced functionality of Netdata Cloud.
+
+## Metadata
+
+The information we store in Netdata Cloud is the following (using the publicly available demo server `frankfurt.my-netdata.io` as an example) :
```
Comment (joelhans) [inline aclk/README.md:143]:
```suggestion
The information we store in Netdata Cloud is the following (using the publicly available demo server `frankfurt.my-netdata.io` as an example):
```

File: aclk/README.md [hunk 6]
```diff
@@ -129,3 +129,28 @@ If you changed the runtime setting in your `var/lib/netdata/cloud.d/cloud.conf`
...
+- User data:
+  - user email
+- For each Netdata Agent node that is connected to Netdata Cloud:
```
Comment (joelhans) [inline aclk/README.md:146]:
```suggestion
- For each node claimed to your Spaces in Netdata Cloud:
```

File: aclk/README.md [hunk 7]
```diff
@@ -129,3 +129,28 @@ If you changed the runtime setting in your `var/lib/netdata/cloud.d/cloud.conf`
...
+
+All the data that the user sees in the web browser when using Netdata Cloud, are actually streamed directly from the Netdata Agent to the Netdata Cloud dashboard. They pass through our systems, but they are not stored.
+
+We do however store a limited number of *metadata* to be able to offer the stunning visualizations and advanced functionality of Netdata Cloud.
+
+## Metadata
+
+The information we store in Netdata Cloud is the following (using the publicly available demo server `frankfurt.my-netdata.io` as an example) :
+- User data:
+  - user email
```
Comment (joelhans) [inline aclk/README.md:145]:
```suggestion
- The email address you used to sign up/or sign in
```

File: aclk/README.md (hunk 2 above)
Comment (joelhans) [inline aclk/README.md:151]: You need to add another two spaces to each of these so they're indented an additional levele.

File: aclk/README.md [hunk 8]
```diff
@@ -129,3 +129,28 @@ If you changed the runtime setting in your `var/lib/netdata/cloud.d/cloud.conf`
...
 
 [![analytics](https://www.google-analytics.com/collect?v=1&aip=1&t=pageview&_s=1&ds=github&dr=https%3A%2F%2Fgithub.com%2Fnetdata%2Fnetdata&dl=https%3A%2F%2Fmy-netdata.io%2Fgithub%2Faclk%2FREADME&_u=MAC~&cid=5792dfd7-8dc4-476b-af31-da2fdb9f93d2&tid=UA-64295674-3)](<>)
+
+# ACLK and privacy
+
+Privacy is very important to us and we firmly believe that the data of our users belong to themselves. This is why **we actually don't store any metric data in the Netdata Cloud**. 
+
+All the data that the user sees in the web browser when using Netdata Cloud, are actually streamed directly from the Netdata Agent to the Netdata Cloud dashboard. They pass through our systems, but they are not stored.
+
+We do however store a limited number of *metadata* to be able to offer the stunning visualizations and advanced functionality of Netdata Cloud.
```
Comment (joelhans) [inline aclk/README.md:139]: Let's keep this content purely factual.

```suggestion
We do however store a limited number of _metadata_ to be able to offer the visualizations and functionality of Netdata Cloud.
```

File: aclk/README.md [hunk 9]
```diff
@@ -129,3 +129,28 @@ If you changed the runtime setting in your `var/lib/netdata/cloud.d/cloud.conf`
 Restart your Agent and [claim your node](/claim/README.md#how-to-claim-a-node).
 
 [![analytics](https://www.google-analytics.com/collect?v=1&aip=1&t=pageview&_s=1&ds=github&dr=https%3A%2F%2Fgithub.com%2Fnetdata%2Fnetdata&dl=https%3A%2F%2Fmy-netdata.io%2Fgithub%2Faclk%2FREADME&_u=MAC~&cid=5792dfd7-8dc4-476b-af31-da2fdb9f93d2&tid=UA-64295674-3)](<>)
+
+# ACLK and privacy
+
+Privacy is very important to us and we firmly believe that the data of our users belong to themselves. This is why **we actually don't store any metric data in the Netdata Cloud**. 
+
+All the data that the user sees in the web browser when using Netdata Cloud, are actually streamed directly from the Netdata Agent to the Netdata Cloud dashboard. They pass through our systems, but they are not stored.
```
Comment (joelhans) [inline aclk/README.md:137]:
```suggestion
All the metric data you see when using Netdata Cloud are streamed from each claimed node. The data then pass through the Netdata Cloud infrastructure, but are not stored, and are finally streamed to your web browser to create visualizations. [Your data is stored](/docs/store/distributed-data-architecture.md ) exclusively inside your network.
```

File: aclk/README.md [hunk 10]
```diff
@@ -129,3 +129,28 @@ If you changed the runtime setting in your `var/lib/netdata/cloud.d/cloud.conf`
...
+  - user email
+- For each Netdata Agent node that is connected to Netdata Cloud:
+ - Node Name
+ - The node information shown in `/api/v1/info`. E.g. `https://frankfurt.my-netdata.io/api/v1/info`
+ - The chart metadata shown in `/api/v1/charts`. E.g. `https://frankfurt.my-netdata.io/api/v1/charts`
+ - The alarm configurations shown in `/api/v1/alarms?all`. E.g. `https://frankfurt.my-netdata.io/api/v1/alarms?all`
+ - Active alarms shown in `/api/v1/alarms`. E.g `https://frankfurt.my-netdata.io/api/v1/alarms`
+
+How we use them:
+- The data are stored in our production database on Google Cloud and some of it is also used in BigQuery, our data lake, for analytics purposes. These analytics are crucial for our product development process.
```
Comment (joelhans) [inline aclk/README.md:154]:
```suggestion
- The data are stored in our production database on Google Cloud. We use some metdata in BigQuery, our data lake, for analytics purposes. These analytics are crucial for our product development process.
```

File: aclk/README.md [hunk 11]
```diff
@@ -129,3 +129,28 @@ If you changed the runtime setting in your `var/lib/netdata/cloud.d/cloud.conf`
...
+- User data:
+  - user email
+- For each Netdata Agent node that is connected to Netdata Cloud:
+ - Node Name
+ - The node information shown in `/api/v1/info`. E.g. `https://frankfurt.my-netdata.io/api/v1/info`
+ - The chart metadata shown in `/api/v1/charts`. E.g. `https://frankfurt.my-netdata.io/api/v1/charts`
+ - The alarm configurations shown in `/api/v1/alarms?all`. E.g. `https://frankfurt.my-netdata.io/api/v1/alarms?all`
+ - Active alarms shown in `/api/v1/alarms`. E.g `https://frankfurt.my-netdata.io/api/v1/alarms`
+
+How we use them:
```
Comment (joelhans) [inline aclk/README.md:153]:
```suggestion
How we use stored metadata:
```

File: aclk/README.md [hunk 12]
```diff
@@ -129,3 +129,28 @@ If you changed the runtime setting in your `var/lib/netdata/cloud.d/cloud.conf`
...
+- The data are stored in our production database on Google Cloud and some of it is also used in BigQuery, our data lake, for analytics purposes. These analytics are crucial for our product development process.
+- Email is used to identify users in regards to product use and to enrich our tools with product use, such as our CRM.
+- This data is only be available to Netdata and never to a 3rd party.
```
Comment (joelhans) [inline aclk/README.md:156]:
```suggestion
- This data is only be available to Netdata, and never to a third party.
```

File: aclk/README.md [hunk 13]
```diff
@@ -129,3 +129,28 @@ If you changed the runtime setting in your `var/lib/netdata/cloud.d/cloud.conf`
...
+
+We do however store a limited number of *metadata* to be able to offer the stunning visualizations and advanced functionality of Netdata Cloud.
+
+## Metadata
+
+The information we store in Netdata Cloud is the following (using the publicly available demo server `frankfurt.my-netdata.io` as an example) :
+- User data:
+  - user email
+- For each Netdata Agent node that is connected to Netdata Cloud:
+ - Node Name
```
Comment (joelhans) [inline aclk/README.md:147]:
```suggestion
 - Hostname (as it appears in Netdata Cloud)
```

File: aclk/README.md [hunk 14]
```diff
@@ -129,3 +129,28 @@ If you changed the runtime setting in your `var/lib/netdata/cloud.d/cloud.conf`
...
+ - The node information shown in `/api/v1/info`. E.g. `https://frankfurt.my-netdata.io/api/v1/info`
```
Comment (joelhans) [inline aclk/README.md:148]:
```suggestion
 - Information shown in `/api/v1/info`. For example: [https://frankfurt.my-netdata.io/api/v1/info](https://frankfurt.my-netdata.io/api/v1/info).
```

File: aclk/README.md [hunk 15]
```diff
@@ -129,3 +129,28 @@ If you changed the runtime setting in your `var/lib/netdata/cloud.d/cloud.conf`
...
+ - The chart metadata shown in `/api/v1/charts`. E.g. `https://frankfurt.my-netdata.io/api/v1/charts`
```
Comment (joelhans) [inline aclk/README.md:149]:
```suggestion
 - Chart metadata shown in `/api/v1/charts`. For example: [https://frankfurt.my-netdata.io/api/v1/charts](https://frankfurt.my-netdata.io/api/v1/charts).

File: aclk/README.md [hunk 16]
```diff
@@ -129,3 +129,28 @@ If you changed the runtime setting in your `var/lib/netdata/cloud.d/cloud.conf`
...
+ - The alarm configurations shown in `/api/v1/alarms?all`. E.g. `https://frankfurt.my-netdata.io/api/v1/alarms?all`
```
Comment (joelhans) [inline aclk/README.md:150]:
```suggestion
 - Alarm configurations shown in `/api/v1/alarms?all`. For example: [https://frankfurt.my-netdata.io/api/v1/alarms?all](https://frankfurt.my-netdata.io/api/v1/alarms?all).
```

File: aclk/README.md (hunk 2 above)
Comment (joelhans) [inline aclk/README.md:151]:
```suggestion
 - Active alarms shown in `/api/v1/alarms`. For example: [https://frankfurt.my-netdata.io/api/v1/alarms](https://frankfurt.my-netdata.io/api/v1/alarms).
```

**Review by joelhans — COMMENTED** [review]
Aside from the suggestions, please move this up the page so it's directly above **Enable and configure the ACLK**.

When you do that, also remove the following text from the introduction:

> Netdata is a distributed monitoring system. Very few data are streamed to the cloud, such as data about:
> - All configured alarms and their current status
> - Metrics that are requested by the cloud user
> - A list of the active collectors

File: aclk/README.md (hunk 2 above)
Comment (joelhans) [reply aclk/README.md:151]: I think they should be. I made them links in my suggestions below.

**Comment by joelhans** [comment]
[Improve docs on Cloud data/privacy policy](https://trello.com/c/RCCn6o9o/341-improve-docs-on-cloud-data-privacy-policy)

**Comment by OdysLam** [comment]
Hey @joelhans, 

Maybe this would be interesting to belong to a security and privacy page, which talks about the whole product, from the agent up to the cloud, from the perspective of privacy and secure (including technical details on how it's secure).

It could even talk about claiming, how it's happening, why it's a secure process, etc.

**Review by aabatangle — DISMISSED** [review]
Agree with Joel's improvements.

**Comment by joelhans** [comment]
> Hey @joelhans,
> 
> Maybe this would be interesting to belong to a security and privacy page, which talks about the whole product, from the agent up to the cloud, from the perspective of privacy and secure (including technical details on how it's secure).
> 
> It could even talk about claiming, how it's happening, why it's a secure process, etc.

Let's talk about this as a second project to come once we merge these improvements. We have lots of disparate security-related information and I agree it would be good to aggregate them into one place, like what we did with the contribute page on Learn.

File: aclk/README.md [hunk 17]
```diff
@@ -14,15 +14,34 @@ through Netdata Cloud. The ACLK establishes an outgoing secure WebSocket (WSS) c
...
 
+## Data privacy
+
+Privacy is very important to us. We firmly believe that your data belongs to you. This is why **we don't store any metric data in Netdata Cloud**. 
+
+All the data that the user sees in the web browser when using Netdata Cloud, are actually streamed directly from the Netdata Agent to the Netdata Cloud dashboard. They pass through our systems, but they are not stored.
+
+We do however store a limited number of *metadata* to be able to offer the stunning visualizations and advanced functionality of Netdata Cloud.
+
+## Metadata
```
Comment (joelhans) [inline aclk/README.md:29]: One last change and then we can merge!

```suggestion
### Metadata
```

**Review by joelhans — CHANGES_REQUESTED** [review]

**Review by aabatangle — DISMISSED** [review]

**Review by joelhans — APPROVED** [review]

**Review by joelhans — APPROVED** [review]

**Review by aabatangle — APPROVED** [review]

**Pull request merged**